src/main/kotlin/car_racing/main.kt

```diff
@@ -0,0 +1,27 @@
+package car_racing
+
+const val VIEW_STRING_CAR = "-"
+const val VIEW_STRING_SEPARATOR = "\n"
+
+fun main() {
+    println("자동차 대수는 몇 대인가요?")
+    val carCount: String = readln()
+    println("시도할 횟수는 몇 회인가요?")
+    val movementTryCount: String = readln()
+    val inputDto = InputDto(carCount, movementTryCount)
+    val randomNumberCarRacingManager = RandomNumberCarRacingManager.fromInputDto(inputDto)
+    val carRacingView = CarRacingView(VIEW_STRING_CAR, VIEW_STRING_SEPARATOR)
+
+    repeat(inputDto.movementTryCount) {
+        setRandomValue(randomNumberCarRacingManager)
+        println(carRacingView.getViewString(randomNumberCarRacingManager))
+        println()
+    }
+}
+
+private fun setRandomValue(randomNumberCarRacingManager: RandomNumberCarRacingManager) {
```

Comment:
해당 함수가 `random`이라는 keyword에 엮여있는 것 같아요! 
만약 랜덤이 아닌 무조건적으로 이동가능한 carRancingManager가 필요할때도 코드 수정을 최소화할 수 있는 네이밍과 구현으로 변경해보면 어떨까요?

```suggestion
private fun setRandomValue(carRacingManager: CarRacingManager) {
```